Fixes the proble where magento returns "Dir XXX is not under storage root path" error when uploading a file.
This is because we have this directory symlinked.

This was introducent in magento 2.2.3 and should be fixed in a future version.

See: https://github.com/magento/magento2/issues/13929

--- a/App/Filesystem/DirectoryResolver.php	2018-06-08 12:33:08.000000000 +0200
+++ b/App/Filesystem/DirectoryResolver.php	2018-06-08 12:33:55.000000000 +0200
@@ -39,7 +39,7 @@
     public function validatePath($path, $directoryConfig = DirectoryList::MEDIA)
     {
         $realPath = realpath($path);
-        $root = $this->directoryList->getPath($directoryConfig);
+        $root = realpath($this->directoryList->getPath($directoryConfig));
         
         return strpos($realPath, $root) === 0;
     }
